### PR TITLE
Add usbguard icon

### LINKED ISSUE #13: Add usbguard icon
Add and install usbguard icon somewhere into /usr/share directory.

### PR DESCRIPTION
**Pull request by ZoltanFridrich**
closes #13

### CODE REVIEW

File: Makefile.am [hunk 1]
```diff
@@ -17,6 +17,9 @@ usbguard_notifier_CXXFLAGS=\
 	@notify_CFLAGS@ \
 	@usbguard_CFLAGS@
 
+appicondir=/usr/share/usbguard-notifier/icons
```
Comment (ZoltanFridrich) [inline Makefile.am]: This path should be modifiable.

Comment (Cropi) [reply Makefile.am]: Does it work? The name of the folder might be **usbguard_notifier.**

Comment (ZoltanFridrich) [reply Makefile.am]: it does work but I am changing this to make it configurable by the user

**Comment by ZoltanFridrich** [comment]
Now the icon path is configurable by setting:
./configure --prefix="icon-path"
Note that macro prefix only sets prefix of the path, like $(prefix)/share/usbguard-notifier [default prefix is /usr/local]

**Pull request merged**